addons/pos_self_order/static/src/kiosk/pages/order_cart/order_cart.xml

```diff
@@ -23,17 +23,17 @@
                                 <div class="product-wrapper d-flex flex-grow-1">
                                     <div class="product-info d-flex flex-row align-items-center justify-content-center flex-grow-1 h-auto">
                                         <div class="d-flex flex-column flex-grow-1 justify-content-center">
-                                            <div class="fs-2 fw-bolder"><span t-esc="line.full_product_name"/></div>
+                                            <div class="fs-2 fw-bolder"><span class="o_kiosk_product_name" t-esc="line.full_product_name"/></div>
 
                                             <div t-foreach="childLines" t-as="cline" t-key="cline.uuid" class="text-muted">
```

Comment:
`cline` is quite a weird name :) `childLine` would really not be that much longer.